server/src/main/java/org/elasticsearch/action/admin/indices/alias/TransportIndicesAliasesAction.java

```diff
@@ -110,8 +110,12 @@ protected void masterOperation(Task task, final IndicesAliasesRequest request, f
             for (final Index index : concreteIndices) {
                 switch (action.actionType()) {
                 case ADD:
+
+                    long now = System.currentTimeMillis();
```

Comment:
I think `long now = System.currentTimeMillis();` should be put before line 110? Otherwise aliases created by an update aliases request can still have different names?